chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -547,10 +546,10 @@ public static DataSet iirLowPassFilterFunction(final DataSet function, final dou
             final double x1 = xValues[i];
             final double y = 0.5 * (yUp[i] + yDown[i]);
             final double mean2 = y * y;
-            final double y2 = 0.5 * Math.pow(ye1[i] + ye2[i], 1);
-            final double avgError2 = Math.abs(y2 - mean2);
-            final double newEYN = Math.sqrt(avgError2 + yen[i] * yen[i]);
-            final double newEYP = Math.sqrt(avgError2 + yep[i] * yep[i]);
+            final double y2 = 0.5 * java.lang.Math.pow(ye1[i] + ye2[i], 1);
+            final double avgError2 = java.lang.Math.abs(y2 - mean2);
+            final double newEYN = java.lang.Math.sqrt(avgError2 + yen[i] * yen[i]);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.sqrt' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)